models/generated/workloads/container.go

```diff
@@ -138,6 +138,11 @@ type Logs struct {
 	Data LogsRedis `bson:"data" json:"data"`
 }
 
+type Stats struct {
+	Type string     `bson:"type" json:"type"`
+	Data StatsRedis `bson:"data" json:"data"`
```

Comment:
the usual pattern when having potential different type of Data is something like this:
```go
type Stats struct {
	Type string    `bson:"type" json:"type"`
	Data json.RawMessage `bson:"data" json:"data"`
}
```
So the Data can be anything. Then you can implement the [json.Unmarshaler](https://godoc.org/encoding/json#Unmarshaler) interface on the Stats type, and you do something like
- decode just the `type` field
- based on the different type you know which type to use to decode the `Data` field

Like this you have a type that is future proof when we will support other endpoint

Reply:
Haaa, that's much better ! Thanks.
I was wondering how to do that, because it seems `Logs` doesn't apply that neither...

Reply:
feel free to open an issue so we also apply this to the Log type